src/accelerate/commands/launch.py

```diff
@@ -307,6 +309,20 @@ def simple_launcher(args):
         mixed_precision = "fp16"
 
     current_env["MIXED_PRECISION"] = str(mixed_precision)
+    if args.num_cpu_threads_per_process == 1:
```

Comment:
We shouldn't change this value if the user has set it to 1. Maybe change the default of `args.num_cpu_threads_per_process` to `None` and then set it to the right default if it's unset.